### DISCUSSION TITLE
Azure SQL - baza demo

### DISCUSSION THREAD

**Discussion by marrybus** [AzureCommunityPL/group#354] [Pomoc] [answered]
hej,

potrzebuje najprostszą bazę Azure SQL - w celach demonstracyjnych - dłuższych niż trial. Wystarczy mi 1 tabelka, kilka rekordów.
Jaka będzie najtańsza opcja? Czy serverless bedzie najtańszą opcją? czy może jednak DTU?

Pozdrawiam,
Mariusz

**Comment by JanuszNowak** [comment, marked as answer]
Napisz kawałek kodu, który tworzy ci bazę nie serwer, potem inseruje wiersze tabelkę lub odzyskuje z backupu. Po czym usuwasz bazę i nie płacisz, a w razie czego w 30 sekund strzyc ci wszystko od nowa.

**Reply by JanuszNowak** [reply]
https://docs.microsoft.com/en-us/azure/azure-sql/database/scripts/create-and-configure-database-powershell

**Reply by marrybus** [reply]
to nie do końca to co potrzebuje, ta baza ma chodzić cały czas.  Ma być demonstracyjną bazą którą będę cyklicznie backupował softem Veeam'a. Czasami może skasuje po to żeby pokazać jak odtworzyć Veeam'em. Od dziś Veeam, oprócz backupu Azure VM, potrafi backupować też Azure SQL.

**Reply by JanuszNowak** [reply]
5 DTU powinno wystarczyć i nie kosztować więc niż 5$ miesięcznie i będzie dostępne 24, serwer less potrafi się pauzować, ale nie jestem pewien, czy faktyczny koszt będzie poniżej 1 $ i dodatkowo dojdzie wybudzanie, które będzie trwało do 60 sekund.

**Reply by JanuszNowak** [reply]
Niestety free tire został usunięty pewnie po tym https://blog.janono.pl/2017/11/azure-sql-ha-8-regions-fail-over-group/ wpisie ;) gdzie Azure sql free tire można było rozpiąć na 8 regionów.

**Reply by marrybus** [reply]
dzięki, $5/miesiąc jest do przełknięcia

**Comment by piotrgwiazda** [comment]
Polecam 5DTU. Znany, nieprzekraczalny koszt.
Serverless ... trzeba to dobrze monitorować. Tam jest jakiś czas
bezczynności rzędu 15 minut i chyba bezczynność oznacza brak połączeń.
Kiedyś mnie to "ugryzło". To chyba bardziej się nadaje jak potrzebujesz
mocną bazę raz w tygodniu a nie na tanie demo.

czw., 30 wrz 2021, 17:40 użytkownik marrybus ***@***.***>
napisał:
